### PR TITLE
Remove trailings spaces from CBM-related asm files

### PR DESCRIPTION
**Pull request by Compyx**
While scanning the mul20/mul40 code I noticed a lot of trailing spaces. So I decided to remove some of those. Currently I've only removed trailing spaces in the CBM files in libsrc/, so should my changes trigger bugs I can at least test/check those bugs.

There's a whole lot more trailing spaces (esp. telestrat), but let's see how this is goes first.

### CODE REVIEW

File: src/cc65/symtab.c [hunk 1]
```diff
@@ -671,7 +671,7 @@ DefOrRef* AddDefOrRef (SymEntry* E, unsigned Flags)
     DOR = xmalloc (sizeof (DefOrRef));
     CollAppend (E->V.L.DefsOrRefs, DOR);
     DOR->Line = GetCurrentLine ();
-    DOR->LocalsBlockId = (long)CollLast (&CurrentFunc->LocalsBlockStack);
+    DOR->LocalsBlockId = (ssize_t)CollLast (&CurrentFunc->LocalsBlockStack);
```
Comment (groessler) [inline src/cc65/symtab.c]: These changes are unrelated to the heading of the PR. And tests failed....

Comment (Compyx) [reply src/cc65/symtab.c]: So how do I fix that?

**Comment by IrgendwerA8** [comment]
>So how do I fix that?

Rollback the data type changing changes?

**Comment by greg-king5** [comment]
I prefer not to have "src/cc65/locals.c" in this PR, at all (unfold the statement to its original two lines).

**Comment by Compyx** [comment]
Alright, I think all traces of my accidental commit have gone now. Sorry about that.

**Comment by oliverschmidt** [comment]
Thanks :-)
